src/Maker/MakeEntity.php

```diff
@@ -490,11 +488,30 @@ private function printAvailableTypes(ConsoleStyle $io): void
             $io->writeln('');
         };
 
+        $printRelationsSection = static function () use ($io) {
```

Comment:
broke out handling relations to reduce the complexity above. otherwise we're nesting if's 3+ layers deep.